src/Psr3ConsoleLogger.php

```diff
@@ -13,14 +13,25 @@ class Psr3ConsoleLogger extends LoggerBase
      * @param mixed $level
      * @param string $message
      * @param array $context
-     * @return null
      */
     public function log($level, $message, array $context = [])
     {
-        echo sprintf(
-            'LOG: [%s] %s',
-            $level,
-            $this->interpolate($message, $context)
-        ) . PHP_EOL;
+        $this->stdout_echo(
+            sprintf(
+                'LOG: [%s] %s',
+                $level,
+                $this->interpolate($message, $context)
+            ) . PHP_EOL
+        );
+    }
+    
+    private function stdout_echo($message)
```

Comment:
Could we rename this to something like `writeToStdOut()`? That would line up better with what it's doing. Including "echo" in the name, especially when we're adding it in order to avoid using `echo`, seems unhelpful.

Reply:
renamed.